README.md

```diff
@@ -1,19 +1,16 @@
-diaspora_ynh
-==========
+Diaspora for YunoHost
 
-[![Integration level](https://dash.yunohost.org/integration/diaspora.svg)](https://dash.yunohost.org/appci/app/diaspora) ![](https://ci-apps.yunohost.org/ci/badges/diaspora.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/diaspora.maintain.svg)[![Shipped version](https://img.shields.io/github/v/release/yunohost-apps/diaspora_ynh)](https://github.com/yunohost-apps/diaspora_ynh/releases)
-
-[![Install diaspora with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=diaspora)
+[![Integration level](https://dash.yunohost.org/integration/diaspora.svg)](https://dash.yunohost.org/appci/app/diaspora) ![](https://ci-apps.yunohost.org/ci/badges/diaspora.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/diaspora.maintain.svg)[![Shipped version](https://img.shields.io/github/v/release/yunohost-apps/diaspora_ynh)](https://github.com/yunohost-apps/diaspora_ynh/releases)  
+[![Install Diaspora with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app=diaspora)
```

Comment:
Oh, I get it. It's the svg logo. 

As a side note, it's often useful to split these kind of PR in several commits. That way, it is much more explicit for the reviewer. For instance, a commit called "Use svg logo" makes it immediately clear what's happening, whereas "fix linter" is much less explicit. (don't bother correcting it for this PR)

Reply:
My apologizes. you're completely right ... I really have to discipline myself to do PR fragmented into specific commits.